models/comments.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+// SELECT
+
+exports.selectCommentsByArticleId = async (article_id) => {
+  const result = await db.query(
+    `SELECT comments.comment_id, comments.body, comments.votes, comments.author, comments.created_at 
```

Comment:
```suggestion
    `SELECT comments.*
```

Reply:
Wouldn't that also select article_id which according to the task description is not on the list of required properties? Yesterday I was told to fix a similar mistake where I selected everything for the articles where the body wasn't required.